src/main.cpp

```diff
@@ -101,13 +102,17 @@ int main() {
     glm::vec3(0.0f, 0.0f, 10.0f),
     Color(0x888888),
     {
-      {Color(0xFFFFFF), glm::vec3(0.5, 0.0, -1.0)}
+      {Color(0xFFDD33), glm::vec3(0.5, 0.0, -1.0)},
```

Comment:
Can we name colours? Just so it's easier for me to map objects on screen that are a certain colour to the code :)